app/controllers/org_admin/phases_controller.rb

```diff
@@ -88,16 +88,21 @@ def create
       phase = Phase.new(phase_params)
       phase.template = Template.find(params[:template_id])
       authorize phase
-      phase = get_new(phase)
+      begin
+        phase = get_new(phase)
 
 # TODO: update this so that the description comes through as part of the normal form attributes [:phase][:description]
-      phase.description = params["phase-desc"]
-      current_tab = params[:r] || 'all-templates'
+        phase.description = params["phase-desc"]
+        current_tab = params[:r] || 'all-templates'
 
-      if phase.save!
-        flash[:notice] = success_message(_('phase'), _('created'))
-      else
-        flash[:alert] = failed_create_error(phase, _('phase'))
+        if phase.save!
+          flash[:notice] = success_message(_('phase'), _('created'))
+        else
+          flash[:alert] = failed_create_error(phase, _('phase'))
+        end
+      rescue StandardError => e
+        flash[:alert] = _('Unable to create a new version of this template.')
```

Comment:
We have specific messages for when an exception is raised. Do we want to display these directly to the user or we want to keep something more generic like this?